src/main/scala/org/broadinstitute/monster/encode/extraction/ExtractionPipeline.scala

```diff
@@ -42,10 +42,10 @@ object ExtractionPipeline {
     /**
       * Generic helper method for extracting linked entities and saving them.
       *
-      * @param encodeEntity the entity from which to extract a field
-      * @param referenceField the field name to use as an identifier in getIds
-      * @param prevData the data to base the ID extraction off of
-      * @param entityFieldName the field name to extract if it isn't @id
+      * @param encodeEntity the entity which should be extracted
+      * @param referenceField the field name to use as an identifier for the encodeEntity (in getIds)
+      * @param prevData the parent entity (which references a set of encodeEntities) from which ID extraction will be based
+      * @param entityFieldName the encodeEntity field name which references the parent (default is @id)
       * @return the extracted linked entities
```

Comment:
@raaidbroad let me know if you have any other thoughts on how these comments could be phrased better